examples/reference/chat/PanelCallbackHandler.ipynb

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
I think it would be very usefull with a link to the LangChain CallbackHandler documentation https://python.langchain.com/docs/modules/callbacks/ such that users understand that this is a general LangChain concept. And such that they have a place to go look for more inspiration.